controllers/absence.controller.js

```diff
@@ -0,0 +1,79 @@
+const catchAsync = require('../utils/catchAsync')
+const ApiError = require('../utils/ApiError')
+const {
+  createAbsence,
+  editAbsence,
+  deleteAbsence,
+  getAbsences
+} = require('../database/models/Absence')
+
+const getAbseneceController = catchAsync(async (req, res) => {
+  const data = await getAbsences()
+  if (data) {
+    res.status(200).send({
+      successmsg: 'Absence retrieved successfully.',
+      data: data
+    })
+  } else {
+    res.status(500).send({
+      msg: 'Unexpected error!!'
+    })
+  }
+})
+
+const editabsenceController = catchAsync(async (req, res) => {
+  const data = await editAbsence()
+  if (data) {
+    res.status(200).send({
+      successmsg: 'Absence details updated successfully',
+      data: data
+    })
+  } else {
+    res.status(500).send({
+      msg: 'Unexpected error!!'
+    })
+    res.status(400).send({
+      msg: 'Invalid input'
+    })
+  }
```

Comment:
We shall not call `.send` twice.
an error will occur. can't send two responses in this approach.